full-stack-tests/core/src/frontend/standalone/GraphicalEditingScope.test.ts

```diff
@@ -23,7 +23,7 @@ function makeUpdate(id: Id64String, range?: Range3d): ElementGeometryChange { re
 function makeDelete(id: Id64String): ElementGeometryChange { return { id, type: DbOpcode.Delete }; }
 
 describe("GraphicalEditingScope", () => {
-  if (ProcessDetector.isElectronAppFrontend) {
+  if (!ProcessDetector.isMobileAppFrontend) {
```

Comment:
core-full-stack-test files containing other tests related to editing that are still only enabled for Electron - should be updated to run for web too?
- BriefcaseTxns.test.ts (for some reason these are `.skip`ped...)
- EditTool.test.ts
- ModelChangeMonitor.test.ts

Reply:
Yes I think they should run now.

Reply:
@calebmshafer skipped the BriefcaseTxns ones in #2430. I'd like to know if they work as expected over web socket.

Reply:
@swbsi I re-enabled BriefcaseTxns tests. They pass in Electron.

Reply:
> 
> 
> @swbsi I re-enabled BriefcaseTxns tests. They pass in Electron.

Some test issues after I enabled for web. I will investigate...

Reply:
Tests should pass now...